packages/authjs-nuxt/src/runtime/lib/client.ts

```diff
@@ -108,6 +108,15 @@ export async function signOut(options?: SignOutParams) {
   if (url.includes("#")) reloadNuxtApp({ persistState: true })
 }
 
+/**
+ * Retrieve the list of providers currently configured
+ * See [OAuth Sign In](https://authjs.dev/guides/basics/pages#oauth-sign-in) for more details.
+ */
+export async function getProviders() {
+  const data = await $fetch<Record<string, any>>("/api/auth/providers")
```

Comment:
Is there a provided type by authjs that we could use instead of any?

Reply:
I am not seeing where they provided a type in the docs. However, it can be assumed that it would be `Provider[]`. Should I type it as such? The thing is, using this type should provide hints for all providers even though all of them wont be used

Reply:
Good point. If it's an array we should at least do `Record<string, unknown>[]`

Reply:
Update the type. Thanks